### PR TITLE
Add bpftrace script to print vtime distributions across DSQs

### PR DESCRIPTION
**Pull request by hodgesds**
For my own better understanding of vtime distribution across dsqs I made this bpftrace script. It might be useful for others as well.

Example output:
```
@dsq_vtime[1]:
[256K, 512K)           1 |                                                    |
[512K, 1M)             0 |                                                    |
[1M, 2M)               0 |                                                    |
[2M, 4M)               7 |                                                    |
[4M, 8M)             118 |                                                    |
[8M, 16M)            194 |                                                    |
[16M, 32M)           335 |                                                    |
[32M, 64M)           474 |                                                    |
[64M, 128M)         1302 |@@                                                  |
[128M, 256M)        1705 |@@                                                  |
[256M, 512M)        1974 |@@@                                                 |
[512M, 1G)          1169 |@@                                                  |
[1G, 2G)            2800 |@@@@                                                |
[2G, 4G)            3454 |@@@@@                                               |
[4G, 8G)            6882 |@@@@@@@@@@@                                         |
[8G, 16G)          30030 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@|
[16G, 32G)         13032 |@@@@@@@@@@@@@@@@@@@@@@                              |

@dsq_vtime[0]:
[0]                 3306 |@@@@@                                               |
[1]                    0 |                                                    |
[2, 4)                 0 |                                                    |
[4, 8)                 0 |                                                    |
[8, 16)                0 |                                                    |
[16, 32)               0 |                                                    |
[32, 64)               0 |                                                    |
[64, 128)              0 |                                                    |
[128, 256)             0 |                                                    |
[256, 512)             0 |                                                    |
[512, 1K)              0 |                                                    |
[1K, 2K)               0 |                                                    |
[2K, 4K)               0 |                                                    |
[4K, 8K)               0 |                                                    |
[8K, 16K)              0 |                                                    |
[16K, 32K)             0 |                                                    |
[32K, 64K)             0 |                                                    |
[64K, 128K)            0 |                                                    |
[128K, 256K)           0 |                                                    |
[256K, 512K)           0 |                                                    |
[512K, 1M)             0 |                                                    |
[1M, 2M)               0 |                                                    |
[2M, 4M)               9 |                                                    |
[4M, 8M)              80 |                                                    |
[8M, 16M)             44 |                                                    |
[16M, 32M)           124 |                                                    |
[32M, 64M)           135 |                                                    |
[64M, 128M)          781 |@                                                   |
[128M, 256M)        1215 |@                                                   |
[256M, 512M)        1350 |@@                                                  |
[512M, 1G)          2415 |@@@                                                 |
[1G, 2G)            5341 |@@@@@@@@                                            |
[2G, 4G)            6570 |@@@@@@@@@                                           |
[4G, 8G)           16110 |@@@@@@@@@@@@@@@@@@@@@@@@                            |
[8G, 16G)          34177 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@|
[16G, 32G)         19574 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                       |
[32G, 64G)            18 |                                                    |
[64G, 128G)           23 |                                                    |
[128G, 256G)          36 |                                                    |
[256G, 512G)          93 |                                                    |
[512G, 1T)           125 |                                                    |

@dsq_vtime[-9223372036854775806]:
[256K, 512K)           1 |                                                    |
[512K, 1M)             1 |                                                    |
[1M, 2M)               2 |                                                    |
[2M, 4M)               2 |                                                    |
[4M, 8M)               3 |                                                    |
[8M, 16M)              3 |                                                    |
[16M, 32M)           147 |                                                    |
[32M, 64M)           159 |                                                    |
[64M, 128M)          160 |                                                    |
[128M, 256M)         402 |                                                    |
[256M, 512M)        1013 |@@                                                  |
[512M, 1G)          1305 |@@@                                                 |
[1G, 2G)            4686 |@@@@@@@@@@@                                         |
[2G, 4G)            6565 |@@@@@@@@@@@@@@@                                     |
[4G, 8G)            9688 |@@@@@@@@@@@@@@@@@@@@@@@                             |
[8G, 16G)           8043 |@@@@@@@@@@@@@@@@@@@                                 |
[16G, 32G)          6457 |@@@@@@@@@@@@@@@                                     |
[32G, 64G)          2831 |@@@@@@                                              |
[64G, 128G)         5294 |@@@@@@@@@@@@                                        |
[128G, 256G)       11676 |@@@@@@@@@@@@@@@@@@@@@@@@@@@                         |
[256G, 512G)       21697 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@|
[512G, 1T)         20861 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@   |

@dsq_vtime[1025]:
[256K, 512K)          41 |                                                    |
[512K, 1M)           110 |                                                    |
[1M, 2M)             234 |                                                    |
[2M, 4M)             496 |                                                    |
[4M, 8M)             975 |@                                                   |
[8M, 16M)           1953 |@@                                                  |
[16M, 32M)          3702 |@@@@                                                |
[32M, 64M)          2987 |@@@                                                 |
[64M, 128M)         5041 |@@@@@@                                              |
[128M, 256M)       24034 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                     |
[256M, 512M)       24486 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                    |
[512M, 1G)         39731 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@|
[1G, 2G)            3086 |@@@@                                                |
[2G, 4G)               0 |                                                    |
[4G, 8G)              77 |                                                    |
[8G, 16G)              0 |                                                    |
[16G, 32G)            10 |                                                    |
[32G, 64G)            85 |                                                    |
[64G, 128G)           75 |                                                    |
[128G, 256G)          84 |                                                    |
[256G, 512G)          18 |                                                    |
```

### CODE REVIEW

**Comment by multics69** [comment]
Thanks @hodgesds ! This looks super useful. I will test this with scx_lavd later today.

**Review by arighi — APPROVED** [review]
This is really useful, thanks! We definitely need more observability scripts.

File: scripts/vtime_dist.bt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	}
+
+	if ($task->scx.dsq->id >= 0) {
+		@dsq_vtime[$task->scx.dsq->id] = hist($task->scx.dsq_vtime);
+	}
+}
+
+
+interval:s:1 {
+    $scx_ops = kaddr("scx_ops");
```
Comment (multics69) [inline scripts/vtime_dist.bt:24]: When I run the script with 6.12-rc2 kernel (CachyOS), it gives the following error message. 

ERROR: Failed to compile: Failed to resolve kernel symbol: scx_ops

If I remove the three slides using scx_ops, it works fine.

Comment (hodgesds) [reply scripts/vtime_dist.bt]: I'll make this optional then so if the script is ran with `./vtime_dist.bt 1` it will print the scheduler.

**Pull request merged**